homeassistant/components/azure_event_hub/__init__.py

```diff
@@ -43,9 +44,9 @@
     {
         DOMAIN: vol.Schema(
             {
+                vol.Required(CONF_EVENT_HUB_INSTANCE_NAME): cv.string,
```

Comment:
This is a breaking change so we can't release this in a patch release unless the integration is already broken for all users. Please add a breaking change paragraph in the PR description.

Reply:
yeah noticed that I put the text in the wrong spot, already fixed.

Reply:
How will this be used for users that don't use the IoTHub?

Reply:
You can choose between using connection string and CONF_EVENT_HUB_INSTANCE_NAME (for IoTHub, but also an option for other use cases), or you can use hub_namespace, CONF_EVENT_HUB_INSTANCE_NAME, and keys. So in both cases you need CONF_EVENT_HUB_INSTANCE_NAME.